docs/gadget-devel/metrics.md

```diff
@@ -93,8 +93,17 @@ you can easily set annotations when running a gadget using the `--annotate` flag
 
 ### Using well-known types in the eBPF code
 
-You can also edit your eBPF source code and use well-known types (TODO links to well known types) instead of annotating
-the individual fields. An event struct using this could look like:
+You can also edit your eBPF source code and use well-known types instead of annotating
+the individual fields. Inspektor Gadget provides the following types:
+
+- `gadget_counter__u32`: a counter of type `__u32`
+- `gadget_counter__u64`: a counter of type `__u64`
+- `gadget_gauge__u32`: a gauge of type `__u32`
+- `gadget_gauge__u64`: a gauge of type `__u64`
+- `gadget_histogram_slot__u32`: a histogram slot of type `__u32`
+- `gadget_histogram_slot__u64`: a histogram slot of type `__u64`
```

Comment:
What do you think about moving this to dedicated section here in the metrics.md file. And then make `docs/gadget-devel/gadget-ebpf-api.md` and this section point to it. This way it'll be easier to maintain as it'll be difficult to remember to update both places if we make a change.